.github/workflows/cypress.yml

```diff
@@ -33,36 +42,51 @@ jobs:
       - name: Install bootstrapper dependencies
         run: pipenv install --dev --deploy
       - run: pipenv run cookiecutter . --no-input -f
+      - uses: actions/upload-artifact@v3
+        with:
+          name: my_project
+          path: my_project/
+          retention-days: 1
+    outputs:
+      BASE_URL: ${{ env.BASE_URL }}
+  Chrome:
+    needs: Setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/download-artifact@v3
+        with:
+          name: my_project
+          path: my_project/
       - name: Install frontend dependencies
         env:
           NPM_CONFIG_PRODUCTION: false
         working-directory: ./my_project/client
         run: npm install
       - name: Run against ${{ needs.Setup.outputs.BASE_URL }}
-        uses: cypress-io/github-action@v2
+        uses: cypress-io/github-action@v4
         with:
           working-directory: my_project/client
           browser: chrome
-          headless: true
         env: 
           NPM_CONFIG_PRODUCTION: false
           CYPRESS_TEST_USER_EMAIL: ${{ secrets.CYPRESS_TEST_USER_EMAIL }}
           CYPRESS_TEST_USER_PASS: ${{ secrets.CYPRESS_TEST_USER_PASS }}
           CYPRESS_baseUrl: ${{ needs.Setup.outputs.BASE_URL }}
   Firefox:
-    runs-on: ubuntu-latest
     needs: Setup
+    runs-on: ubuntu-latest
+    container:
+      image: cypress/browsers:node16.14.2-slim-chrome103-ff102  # https://github.com/cypress-io/cypress-docker-images/tree/master/browsers
+      options: --user 1001
```

Comment:
Firefox should now be more stable when it runs. (lots of questions/confusion/complaints online about running Firefox with Cypress. Hopefully this fix holds for a while)